Tasks/Maven/CodeAnalysis/mavencodeanalysis.ts

```diff
@@ -29,26 +31,22 @@ export function applyEnabledCodeAnalysisGoals(mvnRun: trm.ToolRunner):trm.ToolRu
 // Extract data from code analysis output files and upload results to build server
 export function uploadCodeAnalysisResults():void {
     // Return early if no analysis tools are enabled
-    var enabledCodeAnalysisTools = getEnabledCodeAnalysisTools();
-    if (enabledCodeAnalysisTools.length < 1) {
+    var enabledCodeAnalysisTools: Set<string> = getEnabledCodeAnalysisTools();
```

Comment:
In the future, consider using classes and public / private functions to organise your modules. OO design is easier to maintain in the long run.